pkgs/applications/audio/melmatcheq.lv2/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "MelMatchEQ.lv2";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "brummer10";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s805jgb9msxfq9047s7pxrngizb00w8sm4z94iii80ba65rd20x";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 xorgproto cairo lv2
+  ];
+
+  installFlags = [ "INSTALL_DIR=$(out)/lib/lv2" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/brummer10/MelMatchEQ.lv2";
+    description = "a profiling EQ using a 26 step Mel Frequency Band";
+    maintainers = [ maintainers.magnetophon ];
+    license = licenses.gpl3;
```

Comment:
It's `gpl3` according to [LICENSE](https://github.com/brummer10/MelMatchEQ.lv2/blob/master/LICENSE), but `gpl2Plus` according to [sources](https://github.com/brummer10/MelMatchEQ.lv2/blob/3d91ab9eefe130986d2e6b7700c14726d85d044c/plugin/gx_melmatcheq.h#L6)...

Reply:
Good catch, I opened [an issue](https://github.com/brummer10/MelMatchEQ.lv2/issues/1).

Reply:
Fixed upstream, and PR adapted accordingly.